battlemap_processor/core/battlemap_pipeline.py

```
@@ -0,0 +1,521 @@
+                )
+
+            # Clean up temporary file if it was downloaded
+            if (
+                image_path != img_info.path
+                and self.image_handler.temp_dir  # Check temp_dir is not None
+                and image_path.startswith(self.image_handler.temp_dir)
+            ):
+                try:
+                    os.remove(image_path)
+                except:
+                    pass  # Ignore cleanup errors
```

Ruff (0.12.2)

Use `contextlib.suppress(BaseException)` instead of `try`-`except`-`pass`

(SIM105)